code/drasil-code/lib/Language/Drasil/Choices.hs

```diff
@@ -362,4 +366,27 @@ choicesSent chs = map chsFieldSent [
 
 -- | Helper to combine pairs of 'Sentence's for rendering 'Choices'.
 chsFieldSent :: (Sentence, Sentence) -> Sentence
-chsFieldSent (rec, chc) = rec +:+ S "selected as" +:+. chc
\ No newline at end of file
+chsFieldSent (rec, chc) = rec +:+ S "selected as" +:+. chc
+
+-- | List of user defined function names.
+-- | List is populated with default values.
```

Comment:
Very minor, but I think the second item doesn't need a `|`?

Reply:
Fixed in 4900c9d014d56227f556a60b0e365d3b4bf545c1